implementations/rust/ockam/ockam_node/src/storage/database/migrations/20231006100000_create_database.sql

```diff
@@ -0,0 +1,187 @@
+CREATE TABLE identity
+(
+    identifier     TEXT NOT NULL UNIQUE,
+    change_history TEXT NOT NULL
+);
+
+CREATE TABLE identity_enrollment
+(
+    identifier  TEXT    NOT NULL UNIQUE,
+    enrolled_at INTEGER NOT NULL
+);
+
+CREATE TABLE identity_attributes
+(
+    identifier  TEXT PRIMARY KEY,
```

Comment:
Should it be or should it not be a foreign key? (I really don't know, requires consideration)

Reply:
Maybe, I will leave that for data model improvements.